core/src/main/java/edu/jhuapl/dorset/users/UserService.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2016 The Johns Hopkins University Applied Physics Laboratory LLC
+ * All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package edu.jhuapl.dorset.users;
+
+/**
+ * Services the user data for a Dorset Application.
+ * <p>
+ * The users support information for the Dorset Application and for Agents.
+ */
+public interface UserService {
```

Comment:
This interface needs work. We should enumerate the different kinds of user services and make sure the interface supports them. For example, an OAuth user service might expect one kind of map to users while an LDAP one would have a different.

Also, methods should be consistent. getCurrentUser() returns a string when I would expect it to return a user object.